packages/git/git.3.1.0/opam

```diff
@@ -30,9 +30,9 @@ depends: [
   "mimic"
   "cstruct" {>= "5.0.0"}
   "angstrom" {>= "0.14.0"}
-  "carton"
-  "carton-lwt"
-  "carton-git"
+  "carton" {= "0.1.0"}
```

Comment:
What about using
```suggestion
  "carton" {< "0.2.0"}
```
? There is no point in allowing for a potential bugfix backport in this case?

Reply:
You are right, :+1: fixed!